src/StackAdmin/Azs.Compute.Admin/Module/Azs.Compute.Admin/Generated.PowerShell.Commands/SwaggerPathCommands/Add-AzsVMExtension.ps1

```diff
@@ -125,93 +125,86 @@ function Add-AzsVMExtension {
 
     Process {
 
-
         if ($PSCmdlet.ShouldProcess("$Publisher/$Type/$Version" , "Add virtual machine image extension")) {
 
-            $dontCheck = $true
-            try {
-                $result = Get-AzsVMExtension -Publisher $Publisher -Type $Type -Sku $Sku
-                $dontCheck = $result -eq $null
-            } catch {
-                # No-op
+            if ( -not $PSBoundParameters.ContainsKey('Location')) {
+                $Location = (Get-AzureRMLocation).Location
             }
 
-            if ($dontCheck -or ($Force.IsPresent -or $PSCmdlet.ShouldContinue("Virtual machine image extension exists with values $Publisher/$Type/$Sku, overwrite?", "Performing operation add virtual machine image extension with publisher $Publisher, type $Type, and version $Version."))) {
```

Comment:
same as above, Force flag has to be considered and the prompt.

Reply:
Fixed.

Reply:
same as above,  force flag check needs to happen after the check for Get-AzsVMExtension , if force flag is present, the SDK call should be made again to re-add the vmextension image

Reply:
Not a supported scenario, see above.